nixos/modules/services/monitoring/netdata.nix

```diff
@@ -4,19 +4,20 @@ with lib;
 
 let
   cfg = config.services.netdata;
-
-  configFile = pkgs.writeText "netdata.conf" cfg.configText;
+  localConfig = {
+    global = {
+      "plugins directory" =  "/run/wrappers/bin ${pkgs.netdata}/libexec/netdata/plugins.d";
```

Comment:
Could including `wrappers` directory lead to executable being misclassified as netdata plugins and executed?

Reply:
Plugins for netdata must end with `.plugin` so the likely hood of conflicting with other apps in `wrappers` would be quite low.  

If that's not enough guarantee, the configuration can be called out specifically to only enable the bundled plugins but version bumps for new features in netdata would require config updates from either the user or maintainer here which would cause some churn. 

I think that netdata requiring a `.plugin` extension is enough to keep accidents from happening.

Reply:
what do you think about the following solution?

```diff
╰─ % git diff
diff --git a/nixos/modules/services/monitoring/netdata.nix b/nixos/modules/services/monitoring/netdata.nix
index 5c78f5203e2..43edc7c5e08 100644
--- a/nixos/modules/services/monitoring/netdata.nix
+++ b/nixos/modules/services/monitoring/netdata.nix
@@ -4,9 +4,15 @@ with lib;
 let
   cfg = config.services.netdata;
+
+  wrappedPlugins = pkgs.runCommand "wrapped-plugins" {} ''
+    mkdir -p $out/libexec/netdata/plugins.d
+    ln -s /run/wrappers/bin/apps.plugin $out/libexec/netdata/plugins.d/apps.plugin
+  '';
+
   localConfig = {
     global = {
-      "plugins directory" =  "/run/wrappers/bin ${pkgs.netdata}/libexec/netdata/plugins.d";
+      "plugins directory" =  "${wrappedPlugins}/libexec/netdata/plugins.d ${pkgs.netdata}/libexec/netdata/plugins.d";
     };
   };
   mkConfig = generators.toINI {} (recursiveUpdate localConfig cfg.config);
```

Reply:
I'm ok with that. Are there no concerns about sym linking out of the store? I've been trying to avoid it even though the wrapper set up a few lines down will guarantee it's there.